tensorflow/compiler/mlir/runlit.cfg.py

```diff
@@ -63,7 +63,7 @@
 # Tweak the PATH to include the tools dir.
 llvm_config.with_environment('PATH', config.llvm_tools_dir, append_path=True)
 
-for key in ['HIP_VISIBLE_DEVICES', 'CUDA_VISIBLE_DEVICES',
+for key in ['ROCM_PATH', 'HIP_VISIBLE_DEVICES', 'CUDA_VISIBLE_DEVICES',
```

Comment:
may I ask why this `ROCM_PATH` can do this magic? somehow navi31 cannot load ROCm in mlir front-end?

Reply:
this was the mlir codgen issue, @draganmladjenovic can explain better than I.

See this code block: https://github.com/ROCm/tensorflow-upstream/blob/develop-upstream/third_party/xla/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc#L773-L779

AFAIU, this ROCM_PATH wasn't ever set so the `else` block was always executed but /opt/rocm always had llvm until recently. This code change sets ROCM_PATH at this level